client/components/async-load/index.d.ts

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { Component, ReactNode } from 'react';
+
+declare module 'calypso/components/async-load' {
+	interface AsyncLoadProps {
+		placeholder?: ReactNode;
+		require: any;
+		[ key: string ]: any;
+	}
+
+	export default class AsyncLoad extends Component< AsyncLoadProps, { component: any | null } > {}
```

Comment:
Would be nice if we could avoid `any`s 😞

Reply:
I'll improve it, but the nature of this component is complex